src/pytest_check/__init__.py

```diff
@@ -4,4 +4,4 @@
 
 pytest.register_assert_rewrite("pytest_check.check")
 
-__version__ = "1.0.4"
+__version__ = "1.0.5"
```

Comment:
I had to guess at what's required to cut a new version of `pytest-check`, so I hope this change is enough.